<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1058</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="HeaderWidget" name="headerWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#HeaderWidget {
padding: 2px;
border-style: solid;
border-width: 3px;
border-radius: 7px;
border-color: beige;
}</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="GraphicsView" name="graphicsView">
      <property name="minimumSize">
       <size>
        <width>715</width>
        <height>520</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_New"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionMouse"/>
   <addaction name="actionSelect"/>
   <addaction name="actionScroll"/>
   <addaction name="separator"/>
   <addaction name="actionAddTimeEvent"/>
   <addaction name="actionAddTimeBox"/>
  </widget>
  <action name="action_New">
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="actionAddTimeEvent">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>addTimeEvent</string>
   </property>
   <property name="toolTip">
    <string>Add a graphical TimeEvent</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionAddTimeBox">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>addTimeBox</string>
   </property>
   <property name="toolTip">
    <string>Add a graphical TimeBox</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionSelect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>select</string>
   </property>
  </action>
  <action name="actionScroll">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>scroll</string>
   </property>
  </action>
  <action name="actionMouse">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>mouse</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>GraphicsView.hpp</header>
  </customwidget>
  <customwidget>
   <class>HeaderWidget</class>
   <extends>QWidget</extends>
   <header>HeaderWidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
